Formula/cuba.rb

```diff
@@ -0,0 +1,14 @@
+class Cuba < Formula
+  desc "Library for multidimensional numerical integration"
+  homepage "http://www.feynarts.de/cuba/"
+  url "http://www.feynarts.de/cuba/Cuba-4.2.tar.gz"
+  sha256 "6b75bb8146ae6fb7da8ebb72ab7502ecd73920efc3ff77a69a656db9530a5eef"
+
+  def install
+    # The provided makefile does not support parallel build
+    ENV.deparallelize
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+    pkgshare.install "demo"
+  end
```

Comment:
The formula needs a functional test. Either compile some simple C program and link it with the library, or use an example from the provided source or demos.

Reply:
? Then I don't understand, you asked to remove the test section:

> Please remove tests. We only run tests for security software.

Reply:
We don't run the software's own test suite during compilation. We run our own functional test, during installation (including when we install prebuilt binaries), to test that the formula installed well and is functioning.

Look at how other formulas have a `test do` block (e.g., `mpfr.rb`).